fastlane/lib/fastlane/actions/push_git_tags.rb

```diff
@@ -5,9 +5,13 @@ def self.run(params)
         command = [
           'git',
           'push',
+          params[:remote],
```

Comment:
See https://github.com/fastlane/fastlane/pull/6109/files#r78658241, let's only add this to the array if it's provided and make the option optional 👍